pulpcore/app/management/commands/handle-artifact-checksums.py

```diff
@@ -44,8 +109,29 @@ def _download_artifact(self, artifact, checksum, file_path):
                 break
         return restored
 
+    def _report(self, checksums):
+        if checksums:
+            checksums = checksums.split(",")
+            if "sha256" not in checksums:
+                raise CommandError(_("Checksums must contain sha256"))
+        else:
+            checksums = settings.ALLOWED_CONTENT_CHECKSUMS
+
+        forbidden_checksums = set(constants.ALL_KNOWN_CONTENT_CHECKSUMS).difference(checksums)
+
+        _show_on_demand_content(forbidden_checksums)
```

Comment:
I'd probably stick in a tech preview warning for now especially if this goes out in 3.11:

```suggestion
        self.stderr.write(_("Warning: the handle-artifact-checksums report is in tech preview and may change in the future."))
        _show_on_demand_content(forbidden_checksums)
```

Reply:
I agree, added.